README.md

```diff
@@ -1,2 +1,110 @@
-# Hello-Microverse
-Linters Configuration
+<a name="readme-top"></a>
+<div>
+ <h3><b>Microverse README Template</b></h3>
+</div>
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+- [📖 About the Project](#about-project)
+- [💻 Getting Started](#getting-started)
+ - [Setup](#setup)
+ - [Prerequisites](#prerequisites)
+ - [Install](#install)
+ - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
```

Comment:
- [ ] For professional documentation the table of contents should look like in the README template, kindly check and fix the issue 👇🏽 

| Expected | Yours |
|---|---|
| ![image](https://github.com/AdrianScharff/Hello-Microverse/assets/58642949/042cf500-750b-4f98-8185-4d5c04655392) | ![image](https://github.com/AdrianScharff/Hello-Microverse/assets/58642949/197e74e7-0883-4ea3-915e-921519fc3924) |


It is because you need a space between lines 6 and 7 👇🏽 

![image](https://github.com/AdrianScharff/Hello-Microverse/assets/58642949/66f6d876-6f6b-4fa3-a3e0-06c46b120455)

Reply:
- [ ] As mentioned by the previous reviewer, Your table of content look messy, kindly make sure your README file looks like the Microverse template 

> * [ ]  For professional documentation the table of contents should look like in the README template, kindly check and fix the issue 👇🏽
> 
> Expected	Yours
> ![image](https://user-images.githubusercontent.com/58642949/242429502-042cf500-750b-4f98-8185-4d5c04655392.png)	![image](https://user-images.githubusercontent.com/58642949/242429560-197e74e7-0883-4ea3-915e-921519fc3924.png)
> It is because you need a space between lines 6 and 7 👇🏽
> 
> ![image](https://user-images.githubusercontent.com/58642949/242430342-66f6d876-6f6b-4fa3-a3e0-06c46b120455.png)